public/js/delete-photo.js

```diff
@@ -0,0 +1,31 @@
+window.addEventListener('load', init)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/VinzOo93/P6SnowTricks/pullRequest?prid=8761575)
```suggestion
window.addEventListener("load", init)
```